tests/helpers.jsx

```diff
@@ -172,6 +194,7 @@ module.exports = {
       },
       UserStore: {
```

Comment:
The data has to come from somewhere so we're using a flux pattern here to pass stuff around that doesn't fit nicely into a parent->child cascading property thing